timer/plugin.js

```diff
@@ -35,6 +35,20 @@ export default class CCTimer extends Plugin {
 		connection.connect(() => this.setupLivesplit(), () => ingameDisplay.run());
 	}
 
+	prestart() {
+		//This will not work if defined in main, so it's placed here instead
+		sc.OPTIONS_DEFINITION["keys-reset-splits"] = {
+			cat: sc.OPTION_CATEGORY.CONTROLS,
+			hasDivider: true,
+			header: 'ccTimer',
+			init: {
+				key1: ig.KEY.SEMICOLON,
+				key2: void 0
+			},
+			type: 'CONTROLS',
+		};
```

Comment:
It also doesn't work if there is no input-api. Either add a check for it like in the Jetpack mod or a warning in the displayed description.